go.mod

```diff
@@ -11,14 +11,14 @@ require (
 	github.com/Maldris/mathparse v0.0.0-20170508133428-f0d009a7a773
 	github.com/ethereum/go-ethereum v1.12.0
 	github.com/go-echarts/go-echarts/v2 v2.2.6
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.3.1
 	github.com/jedib0t/go-pretty/v6 v6.4.7
 	github.com/pkg/errors v0.9.1
 	github.com/shopspring/decimal v1.3.1
-	github.com/smartcontractkit/libocr v0.0.0-20230922131214-122accb19ea6
+	github.com/smartcontractkit/libocr v0.0.0-20230925165524-ffa38fe11ef8
```

Comment:
libocr is being bumped here as a consequence of adding chainlink-common as a dependency. The only difference in this version of libocr and the previous version, is an update to the README.md.

Reply:
is this the libocr version also used in chainlink-core?

Reply:
No, core uses a newer version